src/esp/gfx/replay/Player.cpp

```diff
@@ -161,6 +161,18 @@ void Player::applyKeyframe(const Keyframe& keyframe) {
     assetInfos_[assetInfo.filepath] = assetInfo;
   }
 
+  for (const auto& deletionInstanceKey : keyframe.deletions) {
```

Comment:
What if we create an instance and then delete it in the same frame? Wouldn't this "leak" the instance during replay? I suspect this scenario was the reasoning behind my initial ordering of these operations inside `applyKeyframe`.

Reply:
You're right, this scenario would cause an issue.